src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -63,4 +55,29 @@ public void onPlayerDamage(EntityDamageEvent e) {
             }
         }
     }
+
+    @EventHandler
+    public void onHungerDamage(EntityDamageEvent e) {
+        if (Items.FOOD_SYNTHESIZER == null || Items.FOOD_SYNTHESIZER.getItem().isDisabled() || !(e.getEntity() instanceof Player)) {
+            return;
+        }
+
+        if (e.getCause() == EntityDamageEvent.DamageCause.STARVATION) {
+            checkAndConsume((Player) e.getEntity(), null);
+        }
+    }
+
+    public void checkAndConsume(Player p, FoodLevelChangeEvent e) {
+        FoodSynthesizer foodSynth=(FoodSynthesizer)SlimefunItem.getByID(Items.FOOD_SYNTHESIZER.getItemId());
```

Comment:
You can also just do `getItem()` rather than getting the item by id by getting its id...

Reply:
Ill change that, I just copied and didnt really touch existing code